src/authenticationService/index.js

```diff
@@ -11,47 +11,44 @@ export const getCurrentUserUID = () => {
   }
 }
 
-export const getGoogleSignInData = async () => {
+export const getCurrentUser = () => {
   try {
-    await GoogleSignin.configure({
-      webClientId: googleConfig.webClientId
-    })
-    const googleSignInData = await GoogleSignin.signIn()
+    return firebase.auth().currentUser
+  } catch (error) {
+    throw new AuthenticationError('Not logged in.')
+  }
+}
 
-    return googleSignInData
+const authenticate = async (AuthenticationMethod) => {
+  // SigninMethod should be either EmailLogin, GoogleLogin, or any other that we may need.
```

Comment:
Please remove this comment. This should be obvious when reading the code that you need to pass an object with an `authenticate` method (Duck Typing). (Also, it's already outdated. It's referring to a variable that changed its name. That's one of the reasons why comments are bad)